### PR TITLE
chore: mint form

### PR DESCRIPTION
**Pull request by csosamorales**
- [x] Layout
- [x] Context
- [x] Border radius SVG
- [x] Success status when mint
- [x] Card improvement: create component for the mint view as all the cards have the same header
- [ ] Separate body from header to have a clearer code
- [ ] debounce on inputs text

To do separate:
- [ ] Field validation: set which fields are required
- [ ] Color picker component
- [ ] Switch component

Nice to have
- [ ] wallet connection, just the button
- [ ] real mint function

https://user-images.githubusercontent.com/52458215/218130313-f07a718e-a999-47ff-a054-0dbbff4bac00.mov

### CODE REVIEW

**Comment by csosamorales** [comment]
[sc-331]
[sc-330]

File: ui/src/views/mint/form-step/fields/logo/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './logo-field';
```
Comment (zoruka) [inline ui/src/views/mint/form-step/fields/logo/index.ts:1]: The folder `logo` should not be `logo-field` also to keep the pattern?

File: ui/src/views/mint/form-step/fields/verify-nfa-field.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  const handleVerifyNFAChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setVerifyNFA(e.target.checked);
+  };
+
+  return (
+    <Card.Text css={{ p: '$4', textAlign: 'left' }}>
+      {/* TODO replace for grid */}
+      <Flex css={{ gap: '$10' }}>
+        <Grid css={{ rowGap: '$1h' }}>
```
Comment (zoruka) [inline ui/src/views/mint/form-step/fields/verify-nfa-field.tsx:15]: It seems like here could be a single grid, maybe the `Card.Text` could be used as a grid in this case.

File: ui/src/views/mint/form-step/form.utils.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+//TODO create env variable
+const DEFAULT_MAX_FILE_SIZE = 10; // in KB
+
+// The file size must be capped to a size that the contract can handle
```
Comment (zoruka) [inline ui/src/views/mint/form-step/form.utils.ts:4]: Could keep the function comment in the same pattern of the others.

File: ui/src/views/mint/form-step/mint-form.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+export const FormStep = () => {
+  const { prevStep, nextStep } = Stepper.useContext();
+  const { appName, appDescription, domain } = Mint.useContext();
+
+  return (
+    <Card.Container css={{ width: '$107h' }}>
+      <MintCardHeader title="NFA Details" onClickBack={prevStep} />
+      <Card.Body>
+        <Grid css={{ rowGap: '$6' }}>
+          <Grid css={{ rowGap: '$4' }}>
```
Comment (zoruka) [inline ui/src/views/mint/form-step/mint-form.tsx:17]: These elements could be a single grid using grid template and areas.

**Review by zoruka — APPROVED** [review]
Great work!

File: ui/src/views/mint/form-step/mint-form.tsx (hunk 4 above)
Comment (csosamorales) [reply ui/src/views/mint/form-step/mint-form.tsx:17]: will do!

**Pull request merged**